runtime/lib/ttmetal/runtime.cpp

```diff
@@ -72,7 +72,9 @@ std::pair<SystemDesc, DeviceIds> getCurrentSystemDesc() {
   std::vector<::tt::target::ChipCoord> chipCoord = {
       ::tt::target::ChipCoord(0, 0, 0, 0),
   };
-  std::vector<::tt::target::ChipChannel> chipChannel;
+  std::vector<::tt::target::ChipChannel> chipChannel = {
+      ::tt::target::ChipChannel(0, 0),
+  };
```

Comment:
I'm not sure this is correct, the channel encoding will soon change with a change that Jackson is working on anyway.